.gitlab-ci.yml

```
@@ -14,38 +14,34 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
 
-test:lint:kubeval:
-  stage: test
-  image: alpine
-  before_script:
-    - wget https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz
-    - tar xf kubeval-linux-amd64.tar.gz kubeval
-    - mv kubeval /usr/local/bin/kubeval
-  script:
-    - kubeval kubernetes.yaml
-
 deploy:production:
   stage: publish
   only:
     refs:
       - master
   image: debian:buster
+  before_script:
+    - export DOCKER_REPOSITORY=mendersoftware/integration-test-runner
+    - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
+    - export COMMIT_TAG=${DOCKER_TAG}"_"${CI_COMMIT_SHA}
+    - export SERVICE_IMAGE=${DOCKER_REPOSITORY}:${COMMIT_TAG}
```

As we are including the template, you can just do:
```suggestion
  before_script:
    - *export_docker_vars
```

---

Great! Thanks!